doc/explanation/operate-server-installer.rst

```diff
@@ -30,8 +30,8 @@ and :kbd:`space` or :kbd:`Enter` keys and a little typing.
 :kbd:`Home` / :kbd:`End` / :kbd:`Page Up` / :kbd:`Page Down` can be used to
 navigate through long lists more quickly in the usual way.
 
-Running the installer over serial
-=================================
+Running the installer over serial port
```

Comment:
This edit to add "port" to the end sounds a bit awkward to me.   If "over serial" won't work, what do you think about:
```suggestion
Running the installer over a serial port
```